frontend/src/components/@common/Marquee/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Marquee from '~/components/@common/Marquee/Marquee';
```

Comment:
이 경우에는 그냥 상대 경로로 쓰는게 좋지 않을까요?

```suggestion
import Marquee from './Marquee';
```